Multisig/UI/SwiftUI/Settings/Get In Touch/EmailSupportViewController.swift

```diff
@@ -28,10 +28,13 @@ struct EmailSupportViewController: UIViewControllerRepresentable {
         let buildNumber = Bundle.main.infoDictionary?["CFBundleVersion"] as? String ?? "unknown"
         let version = "v\(marketingVersion) (\(buildNumber))"
         let safe = try? App.shared.coreDataStack.viewContext.fetch(Safe.fetchRequest().selected()).first
+        let address = safe?.address ?? "None"
+        let chain = safe != nil ? "\(safe!.chain!.name!) (\(safe!.chain!.id!))" : "None"
 
         let message = """
         Gnosis Safe \(version)
-        Safe address: \(safe?.address ?? "None")
+        Safe address: \(address)
+        Chain: \(chain)
```

Comment:
We decided to also include the id in braces as in: Rinkeby (4)

Reply:
it is included

Reply:
Here I asked a related question https://github.com/gnosis/safe-android/pull/1627#discussion_r685073797